modules/cnf-deploying-the-numa-aware-scheduler.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_module-type: PROCEDURE
+[id="cnf-deploying-the-numa-aware-scheduler_{context}"]
+= Deploying the NUMA-aware secondary pod scheduler
+
+After you have installed the NUMA Resources Operator, configure the pod admittance policy for the required machine profile, create the required machine config pool, and deploy the NUMA-aware secondary scheduler. These steps are described in the following procedure.
```

Comment:
I think you can remove the second sentence, "These steps are described in the following procedure."